README.md

```diff
@@ -1,29 +1,211 @@
 # Quarkus Test Suite
 
+Test Suite for Quarkus integration scenarios. Among the standard Quarkus test features, scenarios hardly rely on [Quarkus-test-framework](https://github.com/quarkus-qe/quarkus-test-framework) features
+
 The test suite includes:
 - Bare metal scenario testing
 - OpenShift scenario testing
+- Native testing
+
+## How-to run Quarkus test suite
+
+Please before start be sure that you have all [prerequisites](#Prerequisites) installed.
+
+As a general rule if you have Docker, Java11 and maven installed in your local environment the only command that you need to run is:
```

Comment:
Pls replace lines 14 - 19 with this text:

Docker, JDK 11+, and Apache Maven 3.8 are the base requirements to tun the test suite using the following command:
```
mvn clen verify
```
Refer to [prerequisites](#Prerequisites) section for more details about the tooling.

Maven commands can be executed from the root directory or from the sub-modules. Additionally, if you want to run a single scenario you can use `-Dit.test=<TEST_CLASS_NAME>` flag.